src/components/coursesList/CoursesList.tsx

```diff
@@ -0,0 +1,107 @@
+import { CourseItem } from "components/courseItem/CourseItem";
+import Loader from "components/Loader/Loader";
+import React, { useEffect, useState } from "react";
+import {
+  Container,
+  CourseList,
+  PaginationButton,
+  PaginationButtonContainer,
+  Title,
+} from "./CoursesList.styled";
+
+type IProps = {
+  isLoading: boolean;
+  courses: [];
+};
+
+export const CoursesList: React.FC<IProps> = ({ isLoading, courses }) => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [currentCourses, setCurrentCourses] = useState(courses.slice(0, 10));
+
+  useEffect(() => {
+    if (currentPage === 1) {
+      setCurrentCourses(courses.slice(0, 10));
+    }
+  }, [courses, currentPage]);
+
+  const getCurrentCourses = (page: number) => {
+    switch (page) {
```

Comment:
Using switch statement is a bad practices because it is hard to add new features

Reply:
Done